api/src/main/java/org/pmiops/workbench/api/RuntimeController.java

```diff
@@ -193,6 +195,16 @@ public ResponseEntity<EmptyResponse> createRuntime(String workspaceNamespace, Ru
               persistentDiskRequest.getLabels(),
               LEONARDO_LABEL_IS_RUNTIME,
               LEONARDO_LABEL_IS_RUNTIME_TRUE));
+      persistentDiskRequest.labels(
+          upsertLeonardoLabel(
+              persistentDiskRequest.getLabels(),
```

Comment:
I don't really like how we're repeatedly fetching and mutating the labels field here.  How about something like:

var labels = getLabels()
upsert()
upsert()
upsert()
setLabels(labels)

Reply:
https://github.com/all-of-us/workbench/pull/8066/commits/9ca9b9158c162ae56620f54bd07d8dbff73d1b07